### DISCUSSION TITLE
Showing Github account was too recently created

### DISCUSSION THREAD

**Discussion by lazydevopsguy** [community/community#51455] [GitHub Education]
Hi!
I tried many time to verify my account but I keep receiving the following message:
```
Your GitHub account was too recently created. Please apply again in a few days.
```
I created a new account 10-15 days ago and started using GitHub. Is there any limitations regarding time after which we can apply for guthub student pack ?

**Comment by Ntshangase** [comment]
Hey,

I don't know of any exact time limits before you can apply for student pack, but it recommended that you keep your GitHub active and try again maybe in a weeks time.

**Comment by Krushna-Nanda** [comment]
Your GitHub account was too recently created. Please apply again in a few days. i cretaed account 3 days ago , why im facing these problem , i provide my verified school identity card plz help

**Reply by GurKeerat789** [reply]
Any luck? I am also facing same problem.

**Reply by Sneh4578d** [reply]
Same issue for me. Please help me on this. 
Did this issue got resolved for you @lazydevopsguy , @Krushna-Nanda
Anyone can help?

**Comment by hangingnraorund** [comment]
same here!

**Comment by beter-bocek** [comment]
same issue. did you find any answer ?

**Comment by nobnbabasd123** [comment]
same here!

**Comment by nobnbabasd123** [comment]
same

**Comment by ahmedjmal2001** [comment]
Same here please help
